src/components/ebay-menu-button/test/test.server.js

```diff
@@ -61,12 +61,6 @@ describe('menu-button', () => {
         expect(getByRole('button')).has.class('expand-btn--small');
     });
 
```

Comment:
Should there be a test case for `prefix-label` as well?

Reply:
We do have one i think. 
Regardless we will update with snapshots at some point.